src/components/tx-flow/flows/RecoveryAttempt/RecoveryAttemptReview.tsx

```diff
@@ -18,9 +18,35 @@ type RecoveryAttemptReviewProps = {
   item: RecoveryQueueItem
 }
 
-const RecoveryAttemptReview = ({ item }: RecoveryAttemptReviewProps) => {
-  const [isPending, setIsPending] = useState(false)
+function useAsyncDispatch<T extends (...args: any) => Promise<any>>(dispatchFn: T) {
```

Comment:
Nice! Shall we move this hook to the globals hook folder? Or shall we wait until we have a similar use case?

Reply:
Moved to `useAsync.ts` and renamed to `useAsyncCallback`.